src/clj/game/cards/events.clj

```diff
@@ -1544,6 +1544,30 @@
                  (doseq [c (-> @state :runner :temp-nvram)]
                    (host state side (get-in @state [:runner :identity]) c {:facedown true}))))}
 
+   "Reboot"
+   (letfn [(install-cards [state side eid card to-install]
+             (if (empty? to-install)
+               (do
+                 (move state side (find-latest state card) :rfg)
+                 (effect-completed state side eid))
+               (let [f (first to-install)]
+                 (system-msg state :runner (str "uses Reboot to install " (card-str state f) " facedown"))
```

Comment:
Can we condense this into a single list of all 5 chosen cards? 5 log messages seems excessive.